tools/wptrunner/wptrunner/executors/executormarionette.py

```diff
@@ -780,6 +780,9 @@ def teardown(self):
             self.logger.warning("Exception during reftest teardown:\n%s" %
                                 traceback.format_exc(e))
 
+    def reset(self):
+        self.implementation.reset(**self.implementation_kwargs)
```

Comment:
This will throw with the external reftest runner, given it expects 0 args (well, 1: `self`).

Reply:
I modeled this after the `setup` method which invokes `self.implementation.setup` in a similar fashion. In the class's constructor, the value is defined as an empty dictionary when the external reftest runner is in use.

Reply:
Oh, right. I can't read. :)